doc/classes/MultiplayerPeerExtension.xml

```diff
@@ -10,58 +10,69 @@
 		<method name="_get_available_packet_count" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the available packet count.
 			</description>
 		</method>
 		<method name="_get_connection_status" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the current state of the connection. See [enum ConnectionStatus].
 			</description>
 		</method>
 		<method name="_get_max_packet_size" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the max packet size.
 			</description>
 		</method>
 		<method name="_get_packet" qualifiers="virtual">
 			<return type="int" />
 			<argument index="0" name="r_buffer" type="const uint8_t **" />
 			<argument index="1" name="r_buffer_size" type="int32_t*" />
 			<description>
+				Returns the packet.
 			</description>
 		</method>
 		<method name="_get_packet_peer" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the ID of the [MultiplayerPeerExtension] who sent the most recent packet.
 			</description>
 		</method>
 		<method name="_get_transfer_channel" qualifiers="virtual const">
 			<return type="int" />
 			<description>
-			</description>
+			Returns the transfer channel.
+		</description>
 		</method>
 		<method name="_get_transfer_mode" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the transfer mode.
 			</description>
 		</method>
 		<method name="_get_unique_id" qualifiers="virtual const">
 			<return type="int" />
 			<description>
+				Returns the ID of this [MultiplayerPeer].
 			</description>
 		</method>
 		<method name="_is_refusing_new_connections" qualifiers="virtual const">
 			<return type="bool" />
 			<description>
+				Returns true if it's refusing new connections.
 			</description>
 		</method>
 		<method name="_is_server" qualifiers="virtual const">
 			<return type="bool" />
 			<description>
+				Returns true if the server is implemented.
```

Comment:
`is_server()` IMO has a better explanation in the existing docs for the MultiplayerAPI, see https://docs.godotengine.org/en/latest/classes/class_multiplayerapi.html#class-multiplayerapi-method-is-server

I would reuse and/or maybe directly link to that:
> Returns true if this MultiplayerAPI's multiplayer_peer is valid and in server mode (listening for connections).